src/VisualStudio/Core/Def/Implementation/ChangeSignature/ChangeSignatureDialogViewModel.cs

```diff
@@ -492,7 +498,7 @@ string ModifierText(string @out = default, string @ref = default, string @in = d
                 }
             }
 
-            public string Type => ParameterSymbol.Type.ToDisplayString(s_parameterDisplayFormat);
+            public string Type => ParameterSymbol.Type.ToDisplayString(ParameterSymbol.NullableAnnotation, s_parameterDisplayFormat);
```

Comment:
this seems unfortunate.  We have the Parameter symbol (say for `string? s`).  If you ask its type, "give me your display stirng" shouldn't that give you back `string?` not `string`?  Clearly the Parameter doesn't think it has the `string` type here.  

Note: this is also out of sync with how we do anything else.  i.e. if you have `int*`, `int[]`, or `int?`, we don't need three annotations passed along to say if this was a pointer, array, or nullable.